modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/EmailAddressResourceImpl.java

```diff
@@ -167,26 +169,28 @@ public EmailAddress patchEmailAddress(
 			Long emailAddressId, EmailAddress emailAddress)
 		throws Exception {
 
-		com.liferay.portal.kernel.model.EmailAddress serviceEmailAddress =
-			_emailAddressService.getEmailAddress(emailAddressId);
+		com.liferay.portal.kernel.model.EmailAddress
+			serviceBuilderEmailAddress = _emailAddressService.getEmailAddress(
+				emailAddressId);
 
-		return _updateEmailAddress(emailAddress, serviceEmailAddress);
+		return _updateEmailAddress(emailAddress, serviceBuilderEmailAddress);
 	}
 
 	@Override
 	public EmailAddress patchEmailAddressByExternalReferenceCode(
 			String externalReferenceCode, EmailAddress emailAddress)
 		throws Exception {
 
-		com.liferay.portal.kernel.model.EmailAddress serviceEmailAddress =
-			_emailAddressService.fetchEmailAddressByExternalReferenceCode(
-				externalReferenceCode, contextCompany.getCompanyId());
+		com.liferay.portal.kernel.model.EmailAddress
+			serviceBuilderEmailAddress =
+				_emailAddressService.fetchEmailAddressByExternalReferenceCode(
+					externalReferenceCode, contextCompany.getCompanyId());
 
-		if (serviceEmailAddress == null) {
+		if (serviceBuilderEmailAddress == null) {
```

Comment:
I don't remember if we already have the getEmailAddressByExternalReferenceCode ... if yes please use it so we can remove the if statement